lib/fuse.c

```diff
@@ -280,7 +280,11 @@ static int fuse_load_so_module(const char *module)
 			tmp, dlerror());
 		goto out_dlclose;
 	}
-	ret = fuse_register_module(module, factory, so);
+	if (*factory == NULL) {
```

Comment:
Isn't this dangerous to do if tmp == NULL? factory has not been initialized at this point.

Reply:
If tmp == NULL the functions returns and this line is not reached.
For some reason github displays the unchanged lines in PRs with an extra indentation in this repository.